frontend-implicaction/src/app/admin/companies/companies.component.ts

```diff
@@ -1,9 +1,24 @@
 import {Component} from '@angular/core';
+import {SidebarService} from '../../shared/services/sidebar.service';
+import {CompaniesFormComponent} from './components/companies-form/companies-form.component';
 
 @Component({
   selector: 'app-companies',
   templateUrl: './companies.component.html',
   styleUrls: ['./companies.component.scss']
 })
 export class CompaniesComponent {
+  constructor(
+    private sidebarService: SidebarService,
+  ) {
```

Comment:
```suggestion
  constructor(private sidebarService: SidebarService) {
```
Comme tu n'as qu'un seul paramètre tu peux tout laisser sur une ligne 😉